packages/toolpad-app/src/components/Home.tsx

```diff
@@ -54,8 +53,8 @@ function CreateAppDialog({ onClose, ...props }: CreateAppDialogProps) {
             try {
               const app = await createAppMutation.mutateAsync([name]);
               window.location.href = `/_toolpad/app/${app.id}/editor`;
-            } catch (err) {
-              setShowNameErrorAlert(true);
+            } catch (error) {
+              // Silently catch
```

Comment:
One way we my be able to avoid this is by running
```tsx
window.location.href = `/_toolpad/app/${app.id}/editor`;
```
in the `onSuccess` handler of `useMutation` and use `createAppMutation.mutate([name]);` without `await` instead